test/extensions/filters/http/lua/lua_filter_test.cc

```diff
@@ -1410,8 +1437,9 @@ TEST_F(LuaHttpFilterTest, ImmediateResponse) {
   setup(SCRIPT);
 
   // Perform a GC and snap bytes currently used by the runtime.
-  config_->runtimeGC();
-  const uint64_t mem_use_at_start = config_->runtimeBytesUsed();
+  SingleScriptConfig* script_config = config_->scriptConfig(GLOBAL_SCRIPT_NAME);
```

Comment:
Seems like we are removing this?

Reply:
```
  config_->runtimeGC();
  const uint64_t mem_use_at_start = config_->runtimeBytesUsed();
```
---->
```
  SingleScriptConfig* script_config = config_->scriptConfig(GLOBAL_SCRIPT_NAME);
  script_config->runtimeGC();
  const uint64_t mem_use_at_start = script_config->runtimeBytesUsed();
```

Reply:
The code hasn't been removed, just because it was split into two sections when diffed.